examples/hellohttp/e2e-test.sh

```diff
@@ -42,20 +42,38 @@ function EXPECT_CONTAINS() {
 }
 
 if [[ -z "${1:-}" ]]; then
-  echo "Usage: $(basename $0) <kubernetes namespace> <language ...>"
+  echo "Usage: $(basename $0) <'remote' or 'local' run> <kubernetes namespace> <language ...>"
```

Comment:
can you add a specific error message for each type of failure (i.e., like in line 65 it prints out "ERROR: Languages not provided!"

Reply:
Done